tests/Aether.Tests/Aether.Testing.fs

```diff
@@ -0,0 +1,212 @@
+﻿module Aether.Testing
+
+open FsCheck
+open Swensen.Unquote
+
+/// Properties that can be used to evaluate the conformance of Lens, Prism, and Isos
+/// to certain invariants.
+module Properties =
+    /// Lens properties
+    [<RequireQualifiedAccess;CompilationRepresentation(CompilationRepresentationFlags.ModuleSuffix)>]
+    module Lens =
```

Comment:
Cool, that seems sensible. It's not as if those are likely to change much in future, so seems good!